[debug] [2025-06-14T12:57:54.512Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-06-14T12:57:57.476Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-06-14T12:57:57.477Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-14T12:57:57.477Z] > authorizing via signed-in user (paulomapurunga@gmail.com)
[debug] [2025-06-14T12:57:57.479Z] Checked if tokens are valid: false, expires at: 1749087879315
[debug] [2025-06-14T12:57:57.480Z] Checked if tokens are valid: false, expires at: 1749087879315
[debug] [2025-06-14T12:57:57.480Z] > refreshing access token with scopes: []
[debug] [2025-06-14T12:57:57.481Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-06-14T12:57:57.481Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-06-14T12:57:57.487Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.6.3) on port 50001
[info] i  dataconnect: downloading dataconnect-emulator-2.6.3... {"metadata":{"emulator":{"name":"dataconnect"},"message":"downloading dataconnect-emulator-2.6.3..."}}
[debug] [2025-06-14T12:57:57.615Z] >>> [apiv2][query] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-macos-v2.6.3 
[debug] [2025-06-14T12:57:58.001Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-06-14T12:57:58.001Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-06-14T12:57:58.011Z] [Firebase Plugin] User found:  paulomapurunga@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-06-14T12:57:58.012Z] Checked if tokens are valid: true, expires at: 1749909477001
[debug] [2025-06-14T12:57:58.013Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-06-14T12:57:58.368Z] <<< [apiv2][status] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-macos-v2.6.3 200
[debug] [2025-06-14T12:57:58.369Z] <<< [apiv2][body] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-macos-v2.6.3 [stream]
[debug] [2025-06-14T12:57:59.209Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-06-14T12:57:59.210Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-06-14T12:58:04.235Z] 'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0614 09:58:04.231639   37835 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0614 09:58:04.231639   37835 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:58:04.235Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-06-14T12:58:04.235Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-06-14T12:58:04.235Z] Starting Data Connect Emulator with command {"binary":"/Users/paulomapurunga/.cache/firebase/emulators/dataconnect-emulator-2.6.3","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/Users/paulomapurunga/Playground/processAI/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/Users/paulomapurunga/.cache/firebase/emulators/dataconnect-emulator-2.6.3\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/Users/paulomapurunga/Playground/processAI/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-06-14T12:58:04.386Z] E0614 09:58:04.384510   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:58:04.384510   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:58:04.388Z] E0614 09:58:04.388572   37839 watch.go:76] watchAllDir unable to walk dir `/Users/paulomapurunga/Playground/processAI/dataconnect`: on path "/Users/paulomapurunga/Playground/processAI/dataconnect" (DirEntry: <nil>): lstat /Users/paulomapurunga/Playground/processAI/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:58:04.388572   37839 watch.go:76] watchAllDir unable to walk dir `/Users/paulomapurunga/Playground/processAI/dataconnect`: on path \"/Users/paulomapurunga/Playground/processAI/dataconnect\" (DirEntry: <nil>): lstat /Users/paulomapurunga/Playground/processAI/dataconnect: no such file or directory\n"}}
[debug] [2025-06-14T12:58:04.393Z] I0614 09:58:04.393342   37839 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 09:58:04.393342   37839 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[error] [Firebase Plugin] error:  {}
[debug] [2025-06-14T12:58:14.390Z] E0614 09:58:14.390211   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:58:14.390211   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:58:24.390Z] E0614 09:58:24.389980   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:58:24.389980   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:58:34.391Z] E0614 09:58:34.390629   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:58:34.390629   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:58:44.390Z] E0614 09:58:44.389882   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:58:44.389882   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:58:54.390Z] E0614 09:58:54.389633   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:58:54.389633   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:59:04.390Z] E0614 09:59:04.389960   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:59:04.389960   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:59:14.390Z] E0614 09:59:14.389543   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:59:14.389543   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:59:24.391Z] E0614 09:59:24.390537   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:59:24.390537   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:59:34.390Z] E0614 09:59:34.389894   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:59:34.389894   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:59:44.391Z] E0614 09:59:44.390418   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:59:44.390418   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T12:59:54.390Z] E0614 09:59:54.390573   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 09:59:54.390573   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:00:04.390Z] E0614 10:00:04.390385   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:00:04.390385   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:00:04.390Z] E0614 10:00:04.390493   37839 watch.go:76] watchAllDir unable to walk dir `/Users/paulomapurunga/Playground/processAI/dataconnect`: on path "/Users/paulomapurunga/Playground/processAI/dataconnect" (DirEntry: <nil>): lstat /Users/paulomapurunga/Playground/processAI/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:00:04.390493   37839 watch.go:76] watchAllDir unable to walk dir `/Users/paulomapurunga/Playground/processAI/dataconnect`: on path \"/Users/paulomapurunga/Playground/processAI/dataconnect\" (DirEntry: <nil>): lstat /Users/paulomapurunga/Playground/processAI/dataconnect: no such file or directory\n"}}
[debug] [2025-06-14T13:00:14.391Z] E0614 10:00:14.390814   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:00:14.390814   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:00:24.391Z] E0614 10:00:24.390962   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:00:24.390962   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:00:28.214Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: undefined
[debug] [2025-06-14T13:00:28.214Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-06-14T13:00:28.214Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-14T13:00:28.215Z] > authorizing via signed-in user (paulomapurunga@gmail.com)
[debug] [2025-06-14T13:00:28.215Z] Checked if tokens are valid: true, expires at: 1749909477001
[debug] [2025-06-14T13:00:28.215Z] [Firebase Plugin] User found:  paulomapurunga@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-06-14T13:00:28.215Z] Checked if tokens are valid: true, expires at: 1749909477001
[debug] [2025-06-14T13:00:28.215Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-06-14T13:00:28.219Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: undefined
[debug] [2025-06-14T13:00:28.220Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-06-14T13:00:28.220Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-14T13:00:28.220Z] > authorizing via signed-in user (paulomapurunga@gmail.com)
[debug] [2025-06-14T13:00:28.220Z] Checked if tokens are valid: true, expires at: 1749909477001
[debug] [2025-06-14T13:00:28.220Z] [Firebase Plugin] User found:  paulomapurunga@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-06-14T13:00:28.220Z] Checked if tokens are valid: true, expires at: 1749909477001
[debug] [2025-06-14T13:00:28.221Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-06-14T13:00:29.054Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-06-14T13:00:29.055Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-06-14T13:00:29.773Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-06-14T13:00:29.773Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-06-14T13:00:34.391Z] E0614 10:00:34.390683   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:00:34.390683   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:00:44.393Z] E0614 10:00:44.392400   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:00:44.392400   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:00:54.279Z] E0614 10:00:54.279062   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:00:54.279062   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:01:04.279Z] E0614 10:01:04.278820   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:01:04.278820   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:01:14.278Z] E0614 10:01:14.278460   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:01:14.278460   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:01:24.279Z] E0614 10:01:24.278834   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:01:24.278834   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:01:34.279Z] E0614 10:01:34.278818   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:01:34.278818   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:01:44.278Z] E0614 10:01:44.277964   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:01:44.277964   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:01:54.277Z] E0614 10:01:54.277281   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:01:54.277281   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:02:04.278Z] E0614 10:02:04.277855   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:02:04.277855   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:02:04.279Z] E0614 10:02:04.278034   37839 watch.go:76] watchAllDir unable to walk dir `/Users/paulomapurunga/Playground/processAI/dataconnect`: on path "/Users/paulomapurunga/Playground/processAI/dataconnect" (DirEntry: <nil>): lstat /Users/paulomapurunga/Playground/processAI/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:02:04.278034   37839 watch.go:76] watchAllDir unable to walk dir `/Users/paulomapurunga/Playground/processAI/dataconnect`: on path \"/Users/paulomapurunga/Playground/processAI/dataconnect\" (DirEntry: <nil>): lstat /Users/paulomapurunga/Playground/processAI/dataconnect: no such file or directory\n"}}
[debug] [2025-06-14T13:02:14.278Z] E0614 10:02:14.277809   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:02:14.277809   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:02:24.280Z] E0614 10:02:24.277713   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:02:24.277713   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:02:34.277Z] E0614 10:02:34.276906   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:02:34.276906   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:02:44.278Z] E0614 10:02:44.277776   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:02:44.277776   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:02:54.277Z] E0614 10:02:54.277500   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:02:54.277500   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:03:04.277Z] E0614 10:03:04.277075   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:03:04.277075   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:03:14.277Z] E0614 10:03:14.277229   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:03:14.277229   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:03:24.276Z] E0614 10:03:24.276519   37839 load.go:23] Could not load config from "/Users/paulomapurunga/Playground/processAI/dataconnect": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0614 10:03:24.276519   37839 load.go:23] Could not load config from \"/Users/paulomapurunga/Playground/processAI/dataconnect\": could not find dataconnect.yaml in `/Users/paulomapurunga/Playground/processAI/dataconnect`\n"}}
[debug] [2025-06-14T13:03:34.295Z] I0614 10:03:34.295535   37839 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:03:34.295535   37839 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-06-14T13:03:34.338Z] I0614 10:03:34.338580   37839 control.go:73] [/emulator/processAI 4968] UpdateResources(): done
Schema: sources: schema/schema.gql [2088B] 
Connector "default": sources: connector/mutations.gql [1233B] connector/queries.gql [1802B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:03:34.338580   37839 control.go:73] [/emulator/processAI 4968] UpdateResources(): done\nSchema: sources: schema/schema.gql [2088B] \nConnector \"default\": sources: connector/mutations.gql [1233B] connector/queries.gql [1802B] \n"}}
[debug] [2025-06-14T13:03:34.363Z] I0614 10:03:34.363774   37839 collector.go:107] schema extensions wrote into "/Users/paulomapurunga/Playground/processAI/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [71531B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:03:34.363774   37839 collector.go:107] schema extensions wrote into \"/Users/paulomapurunga/Playground/processAI/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [71531B] \n"}}
[debug] [2025-06-14T13:03:34.364Z] I0614 10:03:34.363872   37839 load.go:115] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:03:34.363872   37839 load.go:115] Finished reloading\n"}}
[debug] [2025-06-14T13:06:47.210Z] I0614 10:06:47.209881   37839 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:06:47.209881   37839 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-06-14T13:06:47.227Z] I0614 10:06:47.227207   37839 codegen.go:82] [connector "default" javascriptSdk] Generating sources into "/Users/paulomapurunga/Playground/processAI/dataconnect-generated/js/default-connector"
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:06:47.227207   37839 codegen.go:82] [connector \"default\" javascriptSdk] Generating sources into \"/Users/paulomapurunga/Playground/processAI/dataconnect-generated/js/default-connector\"\n"}}
[debug] [2025-06-14T13:06:47.243Z] I0614 10:06:47.243382   37839 collector.go:107] schema extensions wrote into "/Users/paulomapurunga/Playground/processAI/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [71531B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:06:47.243382   37839 collector.go:107] schema extensions wrote into \"/Users/paulomapurunga/Playground/processAI/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [71531B] \n"}}
[debug] [2025-06-14T13:06:47.245Z] I0614 10:06:47.245069   37839 collector.go:107] connector "default" javascriptSdk wrote into "/Users/paulomapurunga/Playground/processAI/dataconnect-generated/js/default-connector"
Generated sources: package.json [860B] README.md [2774B] react/package.json [468B] react/esm/package.json [17B] react/index.cjs.js [0B] react/esm/index.esm.js [0B] esm/index.esm.js [112B] esm/package.json [17B] index.cjs.js [147B] react/index.d.ts [228B] react/README.md [5357B] index.d.ts [246B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:06:47.245069   37839 collector.go:107] connector \"default\" javascriptSdk wrote into \"/Users/paulomapurunga/Playground/processAI/dataconnect-generated/js/default-connector\"\nGenerated sources: package.json [860B] README.md [2774B] react/package.json [468B] react/esm/package.json [17B] react/index.cjs.js [0B] react/esm/index.esm.js [0B] esm/index.esm.js [112B] esm/package.json [17B] index.cjs.js [147B] react/index.d.ts [228B] react/README.md [5357B] index.d.ts [246B] \n"}}
[debug] [2025-06-14T13:06:47.249Z] I0614 10:06:47.249306   37839 installsdk.go:80] Running command: /Users/paulomapurunga/.nvm/versions/node/v20.19.2/bin/npm install file:dataconnect-generated/js/default-connector
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:06:47.249306   37839 installsdk.go:80] Running command: /Users/paulomapurunga/.nvm/versions/node/v20.19.2/bin/npm install file:dataconnect-generated/js/default-connector\n"}}
[debug] [2025-06-14T13:07:50.582Z] I0614 10:07:50.581932   37839 load.go:115] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0614 10:07:50.581932   37839 load.go:115] Finished reloading\n"}}
